### PR TITLE
Allow skipping ECS deployment

### PR DESCRIPTION
**Pull request by kenoir**
Allow skipping ECS deployments with a `--skip-ecs-deploy` flag on the `deploy` command so as to prevent force redeploying if required.

### CODE REVIEW

File: RELEASE.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: minor
+
+Allow skipping ECS deployment
```
Comment (alexwlchan) [inline RELEASE.rst:3]: Include a note about what flag to pass to do this?

File: src/deploy/deploy.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def publish(ctx, image_id, label):
     )
 
 
-def _deploy(project, release_id, environment_id, description, confirm=True):
+def _deploy(project, release_id, environment_id, description, confirm=True, skip_ecs_deploy=False):
```
Comment (alexwlchan) [inline src/deploy/deploy.py:147]: Why is this defaulted? Is there any case where it wouldn't be passed?

**Review by alexwlchan — COMMENTED** [review]
What's the use case? When would we want to update the tags but not trigger a new deployment?

**Comment by kenoir** [comment]
I cannot remember the use case for this, so closing.

**Pull request closed without merging**